.travis.yml

```diff
@@ -15,9 +15,9 @@ matrix:
         - tar xzf swift-4.0-RELEASE-ubuntu14.04.tar.gz
         - export PATH=`pwd`/swift-4.0-RELEASE-ubuntu14.04/usr/bin:"${PATH}"
         - sudo apt-get install -y libssl-dev
-        - wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.12.tar.gz
-        - tar xzf libsodium-1.0.12.tar.gz
-        - cd libsodium-1.0.12
+        - wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz
+        - tar xzf libsodium-1.0.16.tar.gz
+        - cd libsodium-1.0.16
```

Comment:
Ideally we don't compile the library each time, but use a packaged version. However libsodium is not packaged for Ubuntu 14.04, and at the time a third-party repository included an older version that we couldn't use. So while this is ok for now, hopefully Travis provides newer Ubuntu's in the future and we can switch to a packaged version instead.